packages/react-components/src/molecules/__tests__/TagList.test.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import TagList from '../TagList';
+
+it('shows all skills when there are few', () => {
```

Comment:
```suggestion
it('shows all tags when there are few', () => {
```

Reply:
can you also move this into a describe block `when summarized` or so, together with the other test it applies to